src/events/CallResponse.ts

```diff
@@ -0,0 +1,62 @@
+import { AttachmentBuilder } from 'discord.js'
+import type { ArgsOf } from 'discordx'
+import { Discord, On } from 'discordx'
+import * as path from 'path'
+
+// Class for simple call/response type things
+
+type CallResponse = {
+  call: string
+  response?: string
+  responseAttachment?: string
+  cooldown: number // seconds
+  lastUse: number
+  caseSensitive: boolean
+}
+
+@Discord()
+abstract class CallAndResponder {
+  private calls: CallResponse[] = [
+    {
+      call: 'tuturu',
+      responseAttachment: path.join(__dirname, '../../assets/Tuturu.png'),
```

Comment:
This will probably fail, I think we have to do `../../src/` as the __dirname is the build directory